ui/components/ui/show-hide-toggle/show-hide-toggle.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+import IconEye from '../icon/icon-eye';
+import IconEyeSlash from '../icon/icon-eye-slash';
+
+const ShowHideToggle = ({
+  id,
+  shown,
+  onChange,
+  ariaLabelHidden,
+  ariaLabelShown,
```

Comment:
Do you think these aria labels should have defaults? I would imagine that most devs wouldn't need to change them seeing as the component is to specifically communicate show and hide? Could possibly save some cognitive load on the developer using this component?

```
  ariaLabelHidden = 'hidden',
  ariaLabelShown = 'shown',
```

Reply:
They are required because they need to be localized. Unless you think we should bring the `t` context into this? I figured it would be best to avoid a dependence upon Redux state, but the locale is sort of a special case... for some UI components it would seem silly to ask all text to be passed in. But anyway.

Reply:
Ahh makes sense! Hmm I definitely need to do some research into how to best to manage translations in regards to reusable ui components. I think keeping it external makes a lot of sense. Some more overhead for the developer but ensures proper translations and keeps the component light. Nice.